remoteworkitem/remoteworkitem_test.go

```diff
@@ -43,14 +59,15 @@ func TestWorkItemMapping(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if workItem.Fields["system.title"] == nil {
+	if !assert.NotNil(t, workItem.Fields["system.title"]) {
```

Comment:
Sorry for the confusion about the `assert` package. It can be shortened from
```go
if !assert.NotNil(t, workItem.Fields["system.title"]) {
     t.Error("system.title not mapped")
}
```
to
```go
assert.NotNil(t, workItem.Fields["system.title"])
```

Assert will take care of letting the test fail. There's also a [`require`](https://github.com/stretchr/testify#require-package) package with harder conditions that you might want to take a look at if assert is to *soft*. It might be that you have to `glide get` the `require` package.

Quote from the `require` package:
>The require package provides same global functions as the assert package, but instead of returning a boolean > result they terminate current test.
>
>See [t.FailNow](https://golang.org/pkg/testing/#T.FailNow) for details.